.github/workflows/daily-deploy-production.yml

```diff
@@ -57,7 +57,7 @@ jobs:
 
       - name: Get release wait time (scheduled release)
         if: ${{ github.event.schedule != '' }}
-        run: echo 'RELEASE_WAIT=60' >> $GITHUB_ENV
+        run: echo 'RELEASE_WAIT=0' >> $GITHUB_ENV
```

Comment:
I'm wondering if we can add logic inside the [alert](https://github.com/department-of-veterans-affairs/vets-website/blob/8ba1ae6eae6d25b9588d4b4ef058495bbb1ac266/.github/workflows/daily-deploy-production.yml#L109) that updates the messaging to say something like `a few minutes` when the release wait is 0 minutes, so that it's a little more accurate. Maybe we can update the expression in the message from this:

`${{ needs.set-env.outputs.RELEASE_WAIT }}`

to something like:

 `${{ needs.set-env.outputs.RELEASE_WAIT == '0' && 'a few' || needs.set-env.outputs.RELEASE_WAIT }}`
 
 This way, the message will display `coming up in a few minutes` when the release wait is 0.
 
Alternatively, we could add an offset to the release wait in the message, so that it adds the average time it takes to build/deploy to the release wait time. That's probably about 10 minutes as of now, but this could  vary between builds, and change in the future; so that would still have some level of inaccuracy. Not that it's ever been accurate before though.
 
Also, totally unrelated to your PR, but it might make sense for us to change the [default release_wait](https://github.com/department-of-veterans-affairs/vets-website/blob/8ba1ae6eae6d25b9588d4b4ef058495bbb1ac266/.github/workflows/daily-deploy-production.yml#L9) to 0, since if we're dispatching the workflow, there's most likely no need to wait since we know what we're deploying. Ultimately, the release wait functionality could be removed from the workflow since it probably won't be used, but it can't hurt to keep it for now, or at least until most apps in the repo are deploying autonomously.

Reply:
I'm down to do either approach, both sound great to me. Removing the release wait functionality completely would simplify the workflow, some of the logic, and the slack messaging, so those are all pluses. 

I guess for now I'll do you first suggestion of just changing the messaging to say "a few". Should it say "a few" for 0 minutes or for anything less than 5 minutes? It might be strange if it says "a few minutes" for 0 minutes but says "1 minute" for 1 minute haha

Reply:
Although to do my suggestion of checking for a release wait of less than 5 minutes, I think I'll need to change the input to be an integer type. Because right now it looks like it's a free form text field, so an input of `0` probably being interpreted as a `'0'`?

Reply:
That's a good point about using that messaging for times under 5 minutes. That sounds good to me
Yeah I think `fromJson` should do the trick ✅